docs/docs/services/qix-engine/access-control.md

```diff
@@ -1,76 +1,132 @@
+valid rule. In this example they must evaluate to `true` in order for the actions to be granted.
+
+The expression involving `_actions` must be present in order to determine which actions that are granted access. See
+section [Actions](#actions) for more details.
+
+## Rule Files
+
+Rules are defined in text files with one rule on each row. Qlik Associative Engine applies rules in the order they
+appear in the file. Two types of rule files are supported. The _Deny_ rule file and the _Allow_ rule file.
+
+The Deny rule file denies access and is always evaluated first. If any rule in the Deny file evaluates to `true`,
+access is immediately denied for the actions proviced and no further evaluation of rules take place. 
+
+The Allow rule file allows access. If a rule in the Allow file evaluates to `true`, access is granted for the
+actions specified and rule evaluation continues, possibly accumulating access to more actions based on succeeding
+rules.
+
+## Engine Configuration for ABAC
+
+### ABAC-related Command Line Switches
+
+| Switch | Values | Description |
+| ------ | ------ | ----------- |
+| `EnableABAC` | `0` or `1` | Enabling/disabling of ABAC rule evalutation. |
+| `Gen3` | `0` or `1` | Enablig/disabling of server mode. Must be set to `1` if enabling ABAC. |
+| `SystemDenyRulePath` | File path | File path to the _Deny_ rules file. |
+| `SystemAllowRulePath` | File path | File path to the _Allow_ rules file. |
+
+### Example
+
+The following command shows how to start a Qlik Associative Engine instance as a Docker container, enabling ABAC, and
+with the Allow and Deny rules in the `allow.txt` and `deny.txt` files.
 
 ```bash
-docker run
-    -v <host folder>:<container folder>
-    qlikcore/engine:<version> \
-    -S Gen3=1 \
-    -S PersistenceMode=3 \
+docker run -v <host folder path>:<container folder path> qlikcore/engine:<version> \
     -S EnableABAC=1 \
-    -S SystemAllowRulePath=/<container folder>/allow.txt \
-    -S SystemDenyRulePath=/<container folder>/deny.txt
+    -S Gen3=1 \
+    -S SystemAllowRulePath=/<container folder path>/allow.txt \
+    -S SystemDenyRulePath=/<container folder path>/deny.txt
 ```
 
-## Allow versus Deny
+### More Code Examples
 
-In addition to using _Allow_ rules to grant access, the Qlik Associative Engine
-also supports _Deny_ rules to deny access.
+The [core-authorization](https://github.com/qlik-oss/core-authorization) repository contains running code examples on
+how to enable ABAC and how to provide rules to Qlik Associative Engine.
 
-!!! Note
-    _Deny_ rules are always applied first, if none matched
-    it will continue with _Allow_ rules.
+## Qlik Associative Engine Rules Language
+
+The following sections provide a more in-depth documentation of the rules language in Qlik Associative Engine - its
+syntax and semantics.
 
-## Actions
+Rules are built upon three central entities:
```

Comment:
"Entities" may be not be the best word, but "objects" already has meaning in ABAC terminology. Suggestions? Is it useful to describe it like this?

Reply:
:+1: I like "concepts"